micrOS/source/Tasks.py

```diff
@@ -341,6 +341,17 @@ def show(tag):
         for t in tasks:
             output.append(f"{t}: {TaskBase.TASKS[t].out}")
         return '\n'.join(output)
+    
+    @staticmethod
```

Comment:
Use: TaskBase.is_busy(tag) static method directly please. (due to memory constraints ;) I don't like wrappers/reimplementations)

Reply:
Sorry, I did not know about TaskBase.is_busy(). :)

Reply:
I just made the refactor weeks ago, to have this toplevel generic access for this :) so new thing :D but it seems to be useful then :D